birdhouse/env.local.example

```diff
@@ -226,6 +235,35 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 #                                   }
 #"
 
+# Load jobs to automatically deploy the custom notebooks from the specific images
+#
+# Ensure we always use the "latest" version of the "cronjob generation code"
+# Path to a checked out repo of "pavics-jupyter-base" which contains the config required for the cronjob generation
+#CHECKOUT_PAVICS_CONFIG="/path/to/checkout/pavics-jupyter-base"
+#export AUTODEPLOY_EXTRA_REPOS="$AUTODEPLOY_EXTRA_REPOS $CHECKOUT_PAVICS_CONFIG"
+
+# Script for automatic cronjob generation
+#DEPLOY_DATA_SPECIFIC_IMAGE_JOB_ENV="$COMPOSE_DIR/components/scheduler/deploy_data_specific_image_job.env"
+# Config for the generation of cronjobs, found on external repo
+#DEPLOY_DATA_PAVICS_JUPYTER_ENV="$CHECKOUT_PAVICS_CONFIG/scheduler-jobs/deploy_data_pavics_jupyter.env"
+
+# Generates a cronjob for each image found in DOCKER_NOTEBOOK_IMAGES
+#if [ -f "$DEPLOY_DATA_PAVICS_JUPYTER_ENV" -a -f "$DEPLOY_DATA_SPECIFIC_IMAGE_JOB_ENV" ]; then
+#    for i in $( seq 1 $(echo $DOCKER_NOTEBOOK_IMAGES | wc -w)) ; do
```

Comment:
I think it's possible to move this loop into `deploy_data_pavics_jupyter.env` as well so all you're left is
```
if [ -f "$DEPLOY_DATA_PAVICS_JUPYTER_ENV" ]; then
  . $DEPLOY_DATA_PAVICS_JUPYTER_ENV
fi
```
Much cleaner for the user.